bin/git-commit-proxy-version

```diff
@@ -33,7 +33,7 @@ new_proxy_rev="release/$new_proxy_version"
 # Checkout the linkerd2-proxy repo to resolve the new proxy version to a SHA
 # and obtain the commit log since the prior version.
 tmp=$(mktemp -d -t l2-proxy.XXX)
-git clone --depth=100 https://github.com/linkerd/linkerd2-proxy "$tmp"
```

Comment:
Not a blocker but how come we've bumped the depth to 500? To get more commits?

Reply:
Yeah, dependabot.